### ISSUE TITLE
any idea when this will work on LOTV?

### ISSUE THREAD

**Issue by mickeyren** [Kalamity/SC2-MacroTrainer#3]

**Comment by Kalamity** [comment]
Probably Never :)

**Comment by bellaPatricia** [comment]
>:

**Comment by mickeyren** [comment]
:( this was the best macro trainer for sc2 :(

**Comment by mickeyren** [comment]
made sc2 so much more fun to play with!

**Comment by Kalamity** [comment]
Well Ive made a fair bit of progress for hots 3.0. But still lots of messy code to revise/update :(

**Comment by bellaPatricia** [comment]
:+1:

**Comment by bellaPatricia** [comment]
https://github.com/bellaPatricia/AnotherSc2Hack/commit/e1cffb3d4858c6c5959537e0e8eddc567848009d

**Comment by mickeyren** [comment]
@bellaPatricia are those offsets from LOTV?

**Comment by bellaPatricia** [comment]
No, those are HotS offsets! But it's a great step in the correct direction.

**Comment by Kalamity** [comment]
To answer the original question, 25/11/15.

The future of this program is still questionable though.

**Comment by mickeyren** [comment]
woot! your going to continue developing this again @Kalamity ? that is super awesome!!!

**Comment by vitamincpp** [comment]
@Kalamity  Any ideas on how to debug SC2 process? Using [TitanHide](https://bitbucket.org/mrexodia/titanhide), I'm able to attach debugger to SC2 and set execution breakpoint, but when i set memory access/write breakpoint,it crashes..
Any advice would be appricated~

**Comment by Kalamity** [comment]
Try it offline
